db/fixtures/ae_datastore/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/limits.rb

```diff
@@ -0,0 +1,58 @@
+#
+# Description: Get quota values.
+#
+
+def quota_values(model_attribute, tag_name)
+  $evm.log(:info, "Getting Quota Values for Model: #{model_attribute} Tag Name: #{tag_name}")
+  object_value = quota_model_value(model_attribute)
+  tag = @source.tags(tag_name).first
+  tag_value = tag_value(tag_name, tag) || 0
+  final_quota_value(tag_value, object_value)
+end
+
+def quota_model_value(model_attr)
+  parent_model_value(model_attr)
+end
+
+def final_quota_value(tag_quota_value, object_quota_value)
+  val = tag_quota_value.zero? ? object_quota_value : tag_quota_value
+  $evm.log(:info, "Final Quota Value: #{val}")
+  val
+end
+
+def vmdb_object(object, id)
+  $evm.vmdb(object).find_by_id(id)
+end
+
+def parent_model_value(attr)
+  value = $evm.parent[attr].to_i
```

Comment:
@tinaafitz 
Should we check if $evm.parent exists and is not nil, raise an exception if missing.